docs/sources/reference/api/docker_remote_api_v1.10.md

```diff
@@ -1343,4 +1343,4 @@ stdout and stderr on the same socket. This might change in the future.
 To enable cross origin requests to the remote api add the flag
 "--api-enable-cors" when running docker in daemon mode.
 
-    $ docker -d -H="192.168.1.9:2375" --api-enable-cors
+    $ docker daemon -H="192.168.1.9:2375" --api-enable-cors
```

Comment:
Not sure; should the older API docs also be updated? It's a bit of a difficult case; newer docker versions support the older API, whereas older docker versions *won't* support the new `daemon` command.

If the `-d` backwards compatibility stays in (for the time being), I *think* the older API docs should not be updated, or should mention both forms (`-d` / `daemon`).

Probably best to get some feedback from a doc maintainers for this.